src/gui/MainWindow.cc

```diff
@@ -2195,6 +2199,55 @@ void MainWindow::sendToOctoPrint()
 #endif // ifdef ENABLE_3D_PRINTING
 }
 
+void MainWindow::sendToLocalSlicer(void)
+{
+#ifdef ENABLE_3D_PRINTING
+  const QString slicer = QString::fromStdString(Settings::Settings::localSlicerExecutable.value());
+
+ const QString fileFormat = QString::fromStdString(Settings::Settings::octoPrintFileFormat.value());
```

Comment:
it uses the octoprint file format for the local slicer feature. Easiest is perhaps to copy this setting into the local slicer section in Preferences. Alternatively make it a common setting, but then we have to deal with file format support filtering for each print service somewhere..